### ISSUE TITLE
Error: EXDEV: cross-device link not permitted

### ISSUE THREAD

**Issue by cube3power** [balderdashy/sails#3389]
guys there,
i found the error ! at 0.11.3 version in sails new reactdemo
"rename 'D:\antd_demo\reactdemo\node_modules\sails\node_modules\grunt-contrib-watch\node_modules\tiny-lr\node_modules\noptify\node_modules\nopt\node_modules\abbrev' -> 'D:\antd_demo\reactdemo\node_modules\abbrev'"

my evn is win7 64bit,nodejs is install in c: and my project is in D: .
if your "sails new reactdemo" in C: will not appear the error!

**Comment by sailsbot** [comment]
Thanks for posting, @cube3power.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
